README.md

```diff
@@ -197,45 +204,5 @@ If you want to cite our 🤗 Datasets [paper](https://arxiv.org/abs/2109.02846)
     primaryClass={cs.CL},
 }
 ```
-```bibtex
-@software{quentin_lhoest_2021_5639822,
-  author       = {Quentin Lhoest and
-                  Albert Villanova del Moral and
-                  Patrick von Platen and
-                  Thomas Wolf and
-                  Mario Šaško and
-                  Yacine Jernite and
-                  Abhishek Thakur and
-                  Lewis Tunstall and
-                  Suraj Patil and
-                  Mariama Drame and
-                  Julien Chaumond and
-                  Julien Plu and
-                  Joe Davison and
-                  Simon Brandeis and
-                  Victor Sanh and
-                  Teven Le Scao and
-                  Kevin Canwen Xu and
-                  Nicolas Patry and
-                  Steven Liu and
-                  Angelina McMillan-Major and
-                  Philipp Schmid and
-                  Sylvain Gugger and
-                  Nathan Raw and
-                  Sylvain Lesage and
-                  Anton Lozhkov and
-                  Matthew Carrigan and
-                  Théo Matussière and
-                  Leandro von Werra and
-                  Lysandre Debut and
-                  Stas Bekman and
-                  Clément Delangue},
-  title        = {huggingface/datasets: 1.15.1},
-  month        = nov,
-  year         = 2021,
-  publisher    = {Zenodo},
-  version      = {1.15.1},
-  doi          = {10.5281/zenodo.5639822},
-  url          = {https://doi.org/10.5281/zenodo.5639822}
-}
-```
+
+You can also cite the library using the Zenodo DOI (one DOI per version of the library), see the full list [here](https://zenodo.org/search?q=conceptrecid:%224817768%22&sort=-version&all_versions=True).
```

Comment:
Some rephrasing, mentioning "reproducibility" requirements, but feel free to ignore it as well.
```suggestion
If you need to cite a specific version of our 🤗 Datasets library for reproducibility, you can use the corresponding version Zenodo DOI from this [list](https://zenodo.org/search?q=conceptrecid:%224817768%22&sort=-version&all_versions=True).
```